### PR TITLE
[FLi0goYV] Rspec controller tests

### PR DESCRIPTION
**Pull request by sandrely26**
This PR include:
-Integration test for controllers
- voting
- item
- callback
- admin/voting
- admin/result

### CODE REVIEW

File: spec/factories/item.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
   factory :item do
     user
     voting
-    title "burritos"
-    special false
+    title Faker::Lorem.word
+	special false
```
Comment (mumoc) [inline spec/factories/item.rb]: fix the spacing ... mm the linter should have scream here

File: spec/factories/voting.rb [hunk 2]
```diff
@@ -2,5 +2,7 @@
   factory :voting do
     title "breakfast January 1 to 5"
     status "open"
+	created_at Faker::Time.backward(14, :evening)
```
Comment (mumoc) [inline spec/factories/voting.rb]: the same here, why does this line have more spaces??

File: spec/support/controller_macros.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    before(:each) do
+      @request.env["devise.mapping"] = Devise.mappings[:user]
+      sign_in FactoryGirl.create(:user)
+    end
+  end
+
+  def login_admin
+    before(:each) do
+      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @user = User.new
```
Comment (mumoc) [inline spec/support/controller_macros.rb]: does this need to be @user? cant it be just user?

**Comment by sandrely26** [comment]
Changes updated D: 
@mumoc @KarenVentura

**Comment by mumoc** [comment]
LGTM, merge merge!!!

Also, please in the Trello card add the reference for the new Trello card you'll be creating about the pending skipped tests ^^

**Comment by sandrely26** [comment]
Ok, thanks :3

**Pull request merged**